content/english/pygame-pong/activity-2.md

```diff
@@ -6,19 +6,19 @@ draft: false
 ---
 
 # Activity 2
-Great! You just fixed the game's boundary. Now, notice that how everything inside game looks black and white. Let's give it some colors. 
+Great! You just fixed the game's boundary. Now, notice how everything inside the game looks black and white. Let's give it some colors. 
 
 To do this, we need to modify the RGB values in `COLOR` attribute.
-RGB stands for RED, GREEN, BLUE. In computer screen you can modify the color of each pixel by modifying its RGB value. Every color can be represented by combination of different RGB values in it. The values are specified in same order as RGB is written.
+RGB stands for RED, GREEN, BLUE. In a computer screen, you can modify the color of each pixel by modifying its RGB value. Every color can be represented by combination of different RGB values in it. The values are specified in same order as RGB is written.
 
```

Comment:
Can you add a color picker similar to how it's done here: https://workshops.nuevofoundation.org/python-pixel/colors_and_pixels/basic-of-colors/

This will allow the students to explore the different colors and their RGB values.